moveit_ros/planning_interface/test/planning_scene_interface_clearScene.cpp

```diff
@@ -0,0 +1,205 @@
+    collision_objects[1].primitives[0].dimensions[0] = 1.8;
+    collision_objects[1].primitives[0].dimensions[1] = 0.1;
+    collision_objects[1].primitives[0].dimensions[2] = 0.1;
+    collision_objects[1].primitive_poses.resize(1);
+    collision_objects[1].primitive_poses[0].position.x = -2.0;
+    collision_objects[1].primitive_poses[0].position.y = 0.9;
+    collision_objects[1].primitive_poses[0].position.z = 1.8;
+    collision_objects[1].primitive_poses[0].orientation.w = 1.0;
+    collision_objects[1].operation = collision_objects[0].ADD;
+
+    // OBJ 3
+    collision_objects[2].id = "3";
+    collision_objects[2].header.frame_id = "panda_link0";
+    collision_objects[2].primitives.resize(1);
+    collision_objects[2].primitives[0].type = collision_objects[0].primitives[0].BOX;
+    collision_objects[2].primitives[0].dimensions.resize(3);
+    collision_objects[2].primitives[0].dimensions[0] = 0.1;
+    collision_objects[2].primitives[0].dimensions[1] = 2.8;
+    collision_objects[2].primitives[0].dimensions[2] = 0.01;
+    collision_objects[2].primitive_poses.resize(1);
+    collision_objects[2].primitive_poses[0].position.x = 0;
+    collision_objects[2].primitive_poses[0].position.y = 0;
+    collision_objects[2].primitive_poses[0].position.z = 3.0;
+    collision_objects[2].primitive_poses[0].orientation.w = 1.0;
+    collision_objects[2].operation = collision_objects[0].ADD;
+
+    // OBJ 4
+    collision_objects[3].id = "4";
+    collision_objects[3].header.frame_id = "panda_link0";
+    collision_objects[3].primitives.resize(1);
+    collision_objects[3].primitives[0].type = collision_objects[0].primitives[0].BOX;
+    collision_objects[3].primitives[0].dimensions.resize(3);
+    collision_objects[3].primitives[0].dimensions[0] = 1.6;
+    collision_objects[3].primitives[0].dimensions[1] = 0.5;
+    collision_objects[3].primitives[0].dimensions[2] = 1;
+    collision_objects[3].primitive_poses.resize(1);
+    collision_objects[3].primitive_poses[0].position.x = -0.3;
+    collision_objects[3].primitive_poses[0].position.y = 0.9;
+    collision_objects[3].primitive_poses[0].position.z = 2.0;
+    collision_objects[3].primitive_poses[0].orientation.w = 1.0;
+    collision_objects[3].operation = collision_objects[0].ADD;
+
+    // Copy same objects in Attached Collision Objects
+    attached_objects.resize(4);
+    for (int i = 0; i < 4; i++)
+    {
+      attached_objects[i].object = collision_objects[i];
+      attached_objects[i].link_name = "panda_link0";
+    }
+  }
+
+protected:
+  moveit::planning_interface::PlanningSceneInterface planning_scene_interface;
+  std::vector<moveit_msgs::CollisionObject> collision_objects;
+  std::vector<moveit_msgs::AttachedCollisionObject> attached_objects;
+};
+
+// Test 1 : ENV with only collision objects 
+TEST_F(ClearSceneFixture, CollisionObjectClearTest)
+{
+  // Verify the scene is clear 
+  ASSERT_EQ(planning_scene_interface.getObjects().size(), std::size_t(0));
```

Comment:
Apply the following simplification to all similar lines:
```suggestion
  ASSERT_EQ(planning_scene_interface.getObjects().size(), 0ul);
```